### ISSUE TITLE
gtag

### ISSUE THREAD

**Issue by jamieb22** [samie/vaadin-ga-tracker#27]
Are there any plans to switch to use of gtag? Reason being, this plugin doesn't seem to work with G-measurement Id's and Google suppor tfor universal id's is being removed in 2023.

**Comment by samie** [comment]
Yes, indeed should be updated. This is likely a good starting point: https://developers.google.com/analytics/devguides/migration/ua/analyticsjs-to-gtagjs

**Comment by samie** [comment]
@jamieb22 Started working on this. Which Vaadin version are you using?

**Comment by jamieb22** [comment]
I'm using Vaadin 21.0.9. I had implemented a custom solution. Definitely not as comprehensive as this project, but it got me going. I may switch once these changes have been implemented.
